manifests/config.pp

```diff
@@ -87,4 +87,15 @@
       content => template($dns::sysconfig_template),
     }
   }
+
+  # the RedHat SCL packages do not consist of the default rfc1912 zones file
+  # therefore the file is created out of this module
+  if $facts['os']['family'] in ['RedHat'] and $dns::globals::scl {
+    file { "${dns::params::localzonepath}":
+      owner   => root,
+      group   => root,
+      mode    => '0755',
+      content => template('dns/named.rfc1912.zones.erb'),
```

Comment:
Since it's an empty template, you could reduce it to:
```suggestion
      content => '',
```

Or in `params.pp` set it to `'unmanaged'` to skip the file altogether. That would be cleaner IMHO.

Reply:
actually this template should not be empty - I've added the content in the last commit.

Reply:
Given it's a static file now, please move it to `files` and use:
```suggestion
      content => file('dns/named.rfc1912.zones'),
```